.github/workflows/tokamak-public-nightly.yml

```diff
@@ -22,31 +382,54 @@ jobs:
           echo ::set-output name=VERSION::$(jq -r .version ./l2geth/package.json)
           echo ::set-output name=GITSHA::$(echo ${GITHUB_SHA} | cut -c1-8)
 
+      - name: Check Runner Architecture
+        id: runner_info
+        run: |
+          ARCHITECTURE=$(uname -m)
+          if [ "$ARCHITECTURE" = "x86_64" ]; then
+            ARCHITECTURE="amd64"
+          fi
+          if [ "$ARCHITECTURE" = "aarch64" ]; then
+            ARCHITECTURE="arm64"
+          fi
+          echo "::set-output name=architecture::$ARCHITECTURE"
+
       - name: Login to Docker Hub
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKERHUB_ACCESS_TOKEN_USERNAME }}
           password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN_SECRET }}
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-
-      - name: Publish L2Geth
-        uses: docker/build-push-action@v2
-        with:
-          context: .
-          file: ./l2geth/Dockerfile
-          push: true
-          tags: |
-            onthertech/titan-l2geth:${{ steps.extractver.outputs.GITSHA }}
-            onthertech/titan-l2geth:nightly
-            onthertech/titan-l2geth:nightly-${{ steps.extractver.outputs.VERSION }}
+      - name: Build and push
+        run: |
+          docker build --build-arg TARGETARCH=${{ steps.runner_info.outputs.architecture }} --build-arg TARGETOS=linux -f ./l2geth/Dockerfile -t onthertech/titan-l2geth:${{ steps.runner_info.outputs.architecture }} .
+          docker image ls
+          docker push onthertech/titan-l2geth:${{ steps.runner_info.outputs.architecture }}
+
+      # - name: Set up Docker Buildx
```

Comment:
you can use original code. just adding platform: ${{steps.runner_info.outputs.architecture}} !

Reply:
Actually, the original code didnot work. I tested some days ago and have just tested a couple of times1

Reply:
How about this test (https://github.com/tokamak-network/tokamak-titan/actions/runs/6812014172)?
I think you test about the original code at the above. Is correct that there is problem at the build-multi-arch-image?
I think we can use `docker manifest` command instead of `manifest-tool`. It is not need to install. It is command of docker.

Reply:
Yes, it is my code test and there is a problem with build-multi-arch-image. I tried with manifest-tool and docker in my local. I have just triggered another action with docker manifest. I think there is a thing incompatible with buildx.

Reply:
Thank you so much. I have become to know you work very hard for this task more than what I thought.
I have some ideas to resolve this situation. After testing it, I will notice it to you tomorrow.
Thank you always. Have a good night. 🙇🏻‍♂️

Reply:
Thank you very much. 
I am looking forward to you idea. 
G9!

Reply:
https://github.com/tokamak-network/tokamak-titan/commit/d20ec553442c2096cf187cb8569bd85d58316f30
you can do like above!

As you say, buildx(or build push action) make a image included provenance. So it is already manifest list not single manifest. So I add the option not included provenance. It works well.

Additionally, We can update the actions like this. and we can use the `provenance` option in the new version of action.

It's my final suggestions
https://github.com/tokamak-network/tokamak-titan/compare/test_brave...steven-test4

Reply:
Thank you for your suggestion,

I have updated and tested successfully!